collectors/bzimport/tests/test_convertors.py

```diff
@@ -562,6 +564,58 @@ def get_flaw_srtnotes(cls):
         }
         """
 
+    @pytest.mark.parametrize(
+        "hightouch,hightouch_lite,result",
+        [
+            # valid pairs
+            ("", "", Flaw.FlawMajorIncident.NOVALUE),
+            ("?", "?", Flaw.FlawMajorIncident.REQUESTED),
+            ("?", "", Flaw.FlawMajorIncident.REQUESTED),
+            ("", "?", Flaw.FlawMajorIncident.REQUESTED),
+            ("-", "-", Flaw.FlawMajorIncident.REJECTED),
+            ("-", "", Flaw.FlawMajorIncident.REJECTED),
+            ("", "-", Flaw.FlawMajorIncident.REJECTED),
+            ("+", "", Flaw.FlawMajorIncident.APPROVED),
+            ("+", "-", Flaw.FlawMajorIncident.APPROVED),
+            ("", "+", Flaw.FlawMajorIncident.CISA_APPROVED),
+            ("-", "+", Flaw.FlawMajorIncident.CISA_APPROVED),
+            # invalid pairs
+            ("+", "+", Flaw.FlawMajorIncident.INVALID),
+            ("+", "?", Flaw.FlawMajorIncident.INVALID),
+            ("?", "+", Flaw.FlawMajorIncident.INVALID),
+            ("-", "?", Flaw.FlawMajorIncident.INVALID),
+            ("?", "-", Flaw.FlawMajorIncident.INVALID),
+        ],
+    )
+    def test_flags_major_incident(self, hightouch, hightouch_lite, result):
+        """
+        Tests that hightouch and hightouch-lite flags from Bugzilla are correctly
+        converted into major_incident_state in OSIDB.
+        """
+        flaw_bug = self.get_flaw_bug()
+        flaw_bug["flags"] = [
+            {"name": "hightouch", "status": hightouch},
```

Comment:
I am unsure if it is OK to write flags this way because Bugzilla returns flags in the following format:

`"flags": [{"name": "hightouch", "status": "+", "id": 100, "type_id": 5, "is_active": 1, "setter": "user@mail.com", "creation_date": "2020-09-30T08:00:00Z", "modification_date": "2020-09-30T09:00:00Z"}]`

Reply:
I would say that is OK, you are only interested in the `name` and `status` in this case, rest of the data is irrelevant and it doesn't matter whether it is present. 

On the other hand, are both `hightouch` and `hightouch_lite` always present in the `flags`? Not sure right now. If not, it might be a good thing to also supply some test scenario with one of those missing.

Reply:
You are right. I will update it.